clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocol.java

```diff
@@ -78,6 +78,8 @@
     public static final Schema ASSIGNMENT_V0 = new Schema(
             new Field(TOPIC_PARTITIONS_KEY_NAME, new ArrayOf(TOPIC_ASSIGNMENT_V0)),
             new Field(USER_DATA_KEY_NAME, Type.NULLABLE_BYTES));
+    public static final Schema TOPIC_PARTITION_ASSIGNMENT_V0 = new Schema(
+            new Field(TOPIC_PARTITIONS_KEY_NAME, new ArrayOf(TOPIC_ASSIGNMENT_V0)));
```

Comment:
This is my initial attempt to use a schema for member assignments that are serialized/deserialized. Not so sure if this is the right place for defining it, or whether I needed to up the version. Your feedback is appreciated.